projects/mercury/src/metadata/common/LinkedDataAPI.js

```diff
@@ -53,6 +53,26 @@ class LinkedDataAPI {
             .then(normalizeTypes)
             .catch(() => null);
     }
+
+    parseHierachyNodes(json) {
```

Comment:
all the mapping/parsing functions are usually placed in _utils_ classes, maybe this one can go to `vocabularyUtils.js`?